espei/run_espei.py

```diff
@@ -114,14 +115,27 @@ def main():
 
     # run ESPEI fitting
     # create the scheduler if not passed
-    if not args.dask_scheduler:
-        args.dask_scheduler = LocalCluster(n_workers=int(multiprocessing.cpu_count()/2), threads_per_worker=1, processes=True)
-    client = ImmediateClient(args.dask_scheduler)
-    logging.info("Running with dask scheduler: %s [%s cores]" % (args.dask_scheduler, sum(client.ncores().values())))
-    try:
-        logging.info("bokeh server for dask scheduler at localhost:{}".format(client.scheduler_info()['services']['bokeh']))
-    except KeyError:
-        logging.info("Install bokeh to use the dask bokeh server.")
+    if args.scheduler == 'MPIPool':
+        from emcee.utils import MPIPool
+        # code recommended by emcee: if not master, wait for instructions then exit
+        client = MPIPool()
+        logging.info("Using MPIPool on {} MPI ranks".format(client.rank))
```

Comment:
Fix this logging. It only shows the rank of the process, not total. Move it after the exit as to not dirty the stderr